cinema/views.py

```diff
@@ -118,12 +145,16 @@ class OrderPagination(PageNumberPagination):
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(
+    mixins.ListModelMixin, mixins.CreateModelMixin, viewsets.GenericViewSet
+):
     queryset = Order.objects.prefetch_related(
         "tickets__movie_session__movie", "tickets__movie_session__cinema_hall"
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
+    authentication_classes = [TokenAuthentication, ]
+    permission_classes = [IsAuthenticated, ]
```

Comment:
It is a good practice to place authentication_classes and permission_classes inside of a tuple, not list, for better efficiency. Please, reformat your code to ensure that